commands/create-account.js

```diff
@@ -34,6 +42,39 @@ module.exports = {
 async function createAccount(options) {
     await eventtracking.track(eventtracking.EVENT_ID_CREATE_ACCOUNT_START, { nodeUrl: options.nodeUrl });
     // NOTE: initialBalance is passed as part of config here, parsed in middleware/initial-balance
+    // periods are disallowed in top-level accounts and can only be used for subaccounts
+    const splitAccount = options.accountId.split('.');
+
+    const splitMaster = options.masterAccount.split('.');
+    const masterRootTLA = splitMaster[splitMaster.length - 1];
+    if (splitAccount.length === 1) {
+        // TLA (bob-with-at-least-maximum-characters)
```

Comment:
oh I see, we are actually using it with < after. So app.alice.test is actually invalid according to this. Let's update the comment so that it reflects the real situation?

Reply:
no, I believe this is correct. @potatodepaulo can you chime in here